src/applications/vaos/project-cheetah/index.jsx

```diff
@@ -14,10 +21,25 @@ import ConfirmationPage from './components/ConfirmationPage';
 import { selectFeatureProjectCheetah } from '../redux/selectors';
 import ReceivedDoseScreenerPage from './components/ReceivedDoseScreenerPage';
 import ContactFacilitiesPage from './components/ContactFacilitiesPage';
+import { FETCH_STATUS } from '../utils/constants';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 
-export function NewBookingSection({ featureProjectCheetah }) {
+export function NewBookingSection({
+  windowsStatus,
+  featureProjectCheetah,
+  openNewBookingPage,
+  pageChangeInProgress,
+}) {
   const match = useRouteMatch();
   const history = useHistory();
+  const location = useLocation();
+
+  useEffect(
+    () => {
+      openNewBookingPage(history);
+    },
+    [pageChangeInProgress],
+  );
 
```

Comment:
Check eligibility for vaccine each time there is a page change. This covers the user hitting the 'back' button and then returning to the page.